packages/parser/package.json

```diff
@@ -43,18 +43,134 @@
       "require": "./lib/cjs/schemas/index.js",
       "import": "./lib/esm/schemas/index.js"
     },
+    "./schemas/alb": {
+      "require": "./lib/cjs/schemas/alb.js",
+      "import": "./lib/esm/schemas/alb.js"
+    },
     "./schemas/api-gateway": {
       "require": "./lib/cjs/schemas/apigw.js",
       "import": "./lib/esm/schemas/apigw.js"
     },
+    "./schemas/api-gateway-proxy": {
```

Comment:
I would not expose these at this stage - we extracted these schemas in a shared file so that they could be consumed by both API Gw & API Gw v2, but I don't see an use case for customers wanting to extend them.

For now I would leave them behind the scenes and consider adding them if there's demand.